### PR TITLE
add carbon component for coal, gas and oil in 2020 and 2025

### PR DESCRIPTION
**Pull request by JulianGeis**
Adding carbon price in 2020 and 2025 for fossil generators 
- reason: electricity generation from lignite too high and from gas too low in 2020

prices so far:

add ons for specific emissions:

prices now:

difference in electricity generation:
before: 

after:

ariadne plots:
before:

after:

### CODE REVIEW

**Comment by JulianGeis** [comment]
results of 3H runs:

image


secondary_energy_detailed

**Comment by lindnemi** [comment]
handle these prices explicitly in `modify_cost_data` for better model transparency

File: ariadne-data/costs_2020-modifications.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
 technology,parameter,value,unit,source,further description
-gas,fuel,11.2,EUR/MWh_th,Ariadne,
-oil,fuel,22.1982,EUR2020/MWh,Ariadne,"$2020 = 0.8775 EUR2020, 1bbl = 1.6998MWh"
-coal,fuel,5.7048,EUR2020/MWh,Ariadne,"$2020 = 0.8775 EUR2020, 1t = 8.06 MWh"
+gas,fuel,16.15,EUR/MWh_th,Ariadne,"plus carbon add on of 10.1725 €/MWh"
```
Comment (toniseibold) [inline ariadne-data/costs_2020-modifications.csv:2]: fix "further description"

File: ariadne-data/costs_2020-modifications.csv [hunk 2]
```diff
@@ -12,3 +12,5 @@ HVDC overhead,investment,995,EUR2020/MW/km,NEP2021+NEP2023
 HVDC submarine,investment,3234,EUR2020/MW/km,NEP2021
 HVAC overhead,investment,1215,EUR2020/MW/km,NEP2023
 HVDC submarine,investment,3183,EUR2020/MW/km,NEP2023
+lignite,fuel,13.471,EUR/MWh_th,"DIW (2013): Current and propsective costs of electricity generation until 2050, http://hdl.handle.net/10419/80348 , pg. 80 text below figure 10, accessed: 2023-12-14.","Based on IEA 2011 data, 10 USD/t.  (plus carbon ad don of 10.1725 €/MWh"
```
Comment (toniseibold) [inline ariadne-data/costs_2020-modifications.csv:15]: typo "add on"

**Review by toniseibold — COMMENTED** [review]
Only minor changes in the comments needed - other than that ready to merge

**Comment by lindnemi** [comment]
very nice work @JulianGeis , thank you!

**Pull request merged**